tests/expand-manifest.jsonld

```diff
@@ -1826,6 +1826,27 @@
       "purpose": "Verifies that an exception is raised when attempting to redefine @context.",
       "input": "expand/er56-in.jsonld",
       "expectErrorCode": "keyword redefinition"
+    }, {
+      "@id": "#ter57",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "Array not found for @json @set.",
+      "purpose": "The value for a @json @set term MUST be an array.",
+      "input": "expand/er57-in.jsonld",
+      "expectErrorCode": "invalid set or list object"
```

Comment:
I this test is accepted, it should have counterparts with other values for `@type` (`ex:datatype`, `@id`), and without any `@type`. I see no reason why `"@type": "@json"` should behave differently.

But more generally, I don't think that this test should be accepted nor that this behavior should be required. There are many other situations where parts of the context are ignored if the encountered value is not of the expected kind. For example, `@type` coertion is ignored if the value of the property is not a JSON scalar.